src/assets/content/dictionary/adaptable.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: Adaptable
-definition: ''
+definition: 'The ability for a system to be personalized based on user input. Adaptability is one of two types of personalization on the spectrum of adaptation, along with adaptivity.'
 sources: 
 - sourceurl: https://citeseerx.ist.psu.edu/document?repid=rep1&type=pdf&doi=c942f84f37c0b2b548e5e189c267e066701fc285
 perspectives:
-- meaning: a system can be personalized, with user input. Unlike with solely-adaptive systems, the user is able to make changes to the system. Adaptability is one of two types of personalization on the spectrum of adaptation, along with adaptivity
+- meaning: a system that can be personalized, with user input. Unlike with solely-adaptive systems, the user is able to make changes to the system.
```

Comment:
issue: There are two periods at the end of the `meaning`. I suggest to remove the period from your `meaning`, because the automatic string concatenation will already add one.

```suggestion
- meaning: a system that can be personalized, with user input. Unlike with solely-adaptive systems, the user is able to make changes to the system
```